src/sentry/api/endpoints/organization_dashboard_details.py

```diff
@@ -105,17 +121,22 @@ def delete(self, request: Request, organization, dashboard) -> Response:
 
         return self.respond(status=204)
 
+    @extend_schema(
+        operation_id="Edit an Organization's Dashboard",
+        parameters=[GlobalParams.ORG_ID_OR_SLUG, DashboardParams.DASHBOARD_ID],
+        request=DashboardDetailsSerializer,
+        responses={
+            200: DashboardDetailsModelSerializer,
+            400: RESPONSE_BAD_REQUEST,
+            403: RESPONSE_FORBIDDEN,
+            404: RESPONSE_NOT_FOUND,
+        },
+        examples=DashboardExamples.DASHBOARD_PUT_RESPONSE,
+    )
     def put(self, request: Request, organization, dashboard) -> Response:
         """
-        Edit an Organization's Dashboard
-        ```````````````````````````````````
-
         Edit an individual organization's dashboard as well as
         bulk edits on widgets (i.e. rearranging widget order).
```

Comment:
What else can be done by editing widgets. Can you please expand on this?

Reply:
I expanded a bit more on the bulk edits here